book/website/collaboration/ppie-events/ppie-events-tech.md

```diff
@@ -0,0 +1,112 @@
+Conversely, in-person events can make use of local resources to support engagement and may be preferred by others who face barriers to joining an online event.
+
+The integration of both in-person and virtual components ensures that participants can engage in a manner that suits their preferences and circumstances.
+
+The flexibility of hybrid formats recognizes the barriers faced by many people with lived experience. 
+Providing a virtual option can overcome any physical resource barriers presented by in-person only events. 
+This flexibility also accommodates the diverse schedules and responsibilities of potential attendees, and takes steps to acknowledge any additional scheduling challenges that may be faced by people with lived experiences.
+
+Although planning and hosting hybrid events often requires more resources to be successful, by making your event hybrid you can better prioritize accessibility practices and the inclusion of attendees with lived experience.
+This empowers a wide range of participants to contribute and benefit from your event's content and community.
+
+If you are planning a hybrid event, please read The Turing Way's {ref}`Chapter on Hybrid Collaboration<cl-hybrid-collab>` and specifically the {ref}`Subchapter on Guidelines for Hybrid Collaboration<cl-hybrid-collab-guidelines>`.
+
+(cl-ppie-events-calendar-invites)=
+## Using Email Reminders and Calendar Invitations
+Many of your attendees will maintain a very busy schedule and often take part in multiple different activities. 
+This will be true for both your research, clinical, and lived experience attendees - and everyone may benefit from the support of reminders and calendar invitations for your event. 
+
+Calendar invites and email reminders can be especially useful to people with lived experience of health conditions, disabilities, and other experiences that add additional commitments and time constraints to their daily life. 
+
+Consider sending both a calendar placeholder invitation for your event, and email reminders the week before, the week of, and the morning of your event. 
+This can help remind your attendees of their registration and help put the important information about your event at the top of their email inbox.
+
+It can also be helpful to utilize the subject line of your email with important information. 
+Consider including the date and title of your event, in both the calendar invitation and any email reminders that you send. 
+
+```{note}
+The Public and Patient Involvement and Engagement community for Multiple Long-Term Conditions has highlighted the importance of sending out reminder emails the week of the event, and the morning of the event. 
+
+This was because medical appointments had often been scheduled after an event's "Save the Date" had been sent and many community members valued the reminders. 
+```
+
+Ensure that any email address you send invitations from is able to contact your attendees (and not flagged as a spam email account). 
```

Comment:
maybe also the reverse? an email that can be reached by your attendees (i.e. don't use a 'no-reply' address)?